docs/marketplace/user/back-office-user-guides/202106.0/catalog/products/product-options/creating-a-product-option.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Creating product options
```

Comment:
1. Since a BO user can create multiple product options, rename `creating-a-product-option.md` > `creating-product-options.md`.
2. Move the document from `/catalog/products/product-options/creating-a-product-option.md` to `/catalog/product-options/creating-a-product-option.md`. There is no `Products` subsection in the BO.